client/src/styles.css

```diff
@@ -0,0 +1,49 @@
+.headerText {
+  font-size: xx-large;
```

Comment:
I have never seen this before :) Allowed values used to be percentages, px, (r)em, but this one is new to me